packages/node-cli/src/commands/manageaccount/ManageAccountInputHandler.js

```diff
@@ -48,13 +48,22 @@ const ANSWERS_NAMES = {
 
 module.exports = class ManageAccountInputHandler extends BaseInputHandler {
 	constructor(options) {
-        super(options);
-        this._sdkPath = options.sdkPath;
+		super(options);
+		this._sdkPath = options.sdkPath;
 	}
 
 	async getParameters(params) {
+		if (!this._runInInteractiveMode) {
+			return params;
+		}
+		let answers;
 		const authIDActionResult = await getAuthIds(this._sdkPath);
-		let answers = await this._selectAuthID(authIDActionResult.data, prompt);
+		if (params.hasOwnProperty(COMMAND.OPTIONS.INFO)) {
+			answers = this._extractAuthIDInfo(authIDActionResult.data, params[COMMAND.OPTIONS.INFO]);
+			delete params[COMMAND.OPTIONS.INFO];
```

Comment:
As this shouldn't be happening unless we do some more modifications, I think it will be better to remove it, and probably also the _extractAuthIDInfo method (If Carlos agree with the "non-bonus solution").